src/image.tsx

```diff
@@ -239,6 +237,7 @@ const Image = React.forwardRef<HTMLImageElement, ImageProps>(({aspectRatio = '1:
             aspectRatio={ratio}
             width={props.width}
             height={props.height}
+            dataAttributes={props.dataAttributes}
```

Comment:
Set a default value for the `data-component-name`:

`dataAttributes={{'component-name': 'Image', ...dataAttributes}}`

Reply:
fixed